RPIPs/RPIP-4.md

```diff
@@ -43,9 +43,13 @@ The snapshot vote SHALL run for at least 7 days -- with 14 days being RECOMMENDE
 
 ### Snapshot Vote Strategy
 
+Eligibility, delegation state, and vote power are all determined based on a snapshot when a vote is created.
+
+The value of RPL used will be the price according to the protocol; at this time, that is embodied in `RocketNetworkPrices.getRPLPrice()`.
+
 #### Eligibility
 
-To be eligible for vote participation, the RPL must be effectively staked in the Rocket Pool protocol as reported by `getNodeEffectiveRPLStake()`.
+All nodes are eligible to receive vote power. For nodes eligible to receive vote power, RPL staked on a node contributes to vote power up to 150% of the value of the node's bonded ETH.
```

Comment:
Can we just delete "For nodes eligible to receive vote power," as confusing/redundant here? Maybe the whole sentence can be redone: "A node's vote power is based on the RPL staked on the node up to 150% of the ..."

Reply:
I don't think there's a significant difference and this is the way it was voted in